.github/workflows/UnitTests.yml

```diff
@@ -36,5 +43,6 @@ jobs:
       uses: julia-actions/julia-buildpkg@v1
 
     - name: Run Unit Tests
-      run: julia --project -e 'using Pkg; Pkg.test()'
-      continue-on-error: ${{ matrix.version == 'nightly' }}
+      uses: julia-actions/julia-runtest@v1
+      with:
+        annotate: true
```

Comment:
Can you please add a carriage return here at the end of this line?

Reply:
```suggestion
        annotate: true

```